jni/nnstreamer-edge.mk

```diff
@@ -0,0 +1,26 @@
+# This mk file defines common features to build NNStreamer library for Android.
+
+ifndef NNSTREAMER_EDGE_ROOT
+$(error NNSTREAMER_EDGE_ROOT is not defined!)
+endif
+
+NNSTREAMER_EDGE_VERSION := 0.2.1
```

Comment:
IMO nns-edge version is unnecessary in android build. use nnstreamer version only?)